webpack.js

```diff
@@ -17,4 +17,8 @@ webpackConfig.module = {
 	]
 }
 
+webpackConfig.resolve.alias = {
+  MaterialIcons: path.resolve(__dirname, 'img/material'),
+}
```

Comment:
We could probably make use of https://webpack.js.org/api/module-methods/#webpackchunkname to put all material icons into one chunk so we avoid requesting one js file for each of the icons